Umbraco-Cloud/Security/index.md

```diff
@@ -85,6 +85,27 @@ In the future, the cookie will be set to `HttpOnly` on Umbraco Cloud to conform
 
 For more information see [the related GitHub issue](https://github.com/Azure/app-service-announcements/issues/12).
 
+## Blacklist specific IPs from accessing you website
```

Comment:
As we prefer not to use the word "blacklist", I've tried to rewrite this heading.

```suggestion
## Website access based on specific IPs
```

Reply:
Maybe "Deny specific IPs from accessing your website" instead?

Reply:
Yes, that's also a good option! 👍